src/app/helpers/externalUrlHelper.js

```diff
@@ -0,0 +1,89 @@
+/**
```

Comment:
Similar API is there in content-service also. why is this implementation in sunbird-portal?